Jenkinsfile.release.prepare

```
@@ -237,13 +241,17 @@ String getRepoName(String repo) {
     return (repo.find(':')) ? repo.split(':')[0] : repo
 }
 
-List collectRepos(List repos, String branch) {
+List collectRepos(List repos, String branch, String jenkins_config_path = '') {
```

```suggestion
Map collectRepos(List repos, String branch, String jenkins_config_path = '') {
```
the definition seems to be wrong according to the returned value

---

nope, it returns `repos.collect{}` so a List :)